src/shogun/labels/BinaryLabels.cpp

```diff
@@ -54,7 +54,7 @@ CBinaryLabels::CBinaryLabels(SGVector<float64_t> src, float64_t threshold) : CDe
 	SGVector<float64_t> labels(src.vlen);
 	for (int32_t i = 0; i < labels.vlen; i++)
 	{
-		labels[i] = src[i] + threshold >= 0 ? +1.0 : -1.0;
+		labels[i] = src[i] >= threshold ? +1.0 : -1.0;
```

Comment:
It is called in apply_binary, but I set the default threshold value to 0, as it initially was, so it shouldn't be affecting the existing code.